symbulate/distributions.py

```diff
@@ -536,12 +567,12 @@ def __init__(self, dfN, dfD):
         if dfN > 0:
             self.dfN = dfN
         else:
-            raise Exception("Degrees of freedom in numerator cannot be less than or equal to 0")
+            raise Exception("Degrees of freedom in numerator must be greater than 0")
```

Comment:
I would write this as "dfN (numerator degrees of freedom) must be greater than 0".

Ditto for dfD below.